usr/share/rear/prep/default/310_include_uefi_tools.sh

```diff
@@ -5,10 +5,13 @@ if grep -qw 'noefi' /proc/cmdline; then
     return
 fi
 
-# next step, is checking /boot/efi directory case-insensitive for the /EFI part (we need it)
-if [[ $(find /boot -maxdepth 1 -iname efi -type d | wc -l) -eq 0 ]] ; then
-    return    # must be mounted
-fi
+# Next step, is checking /boot/efi directory case-insensitive for the /EFI part (we need it).
+# If no /boot/[eE][fF][iI] directory can be found we cannot copy the UEFI binaries we might need.
```

Comment:
First and foremost:
I am not at all a (U)EFI expert.
I only looked at the code and to me it looked "strange".

Regarding a proper solution:
Perhaps even mount the EFI System Partition
if it is not yet mounted when things from there
are needed.
Perhaps one can ignore whether or not
the EFI System Partition is already mounted
and mount it (again) in any case at a ReaR specific
mount point?